### ISSUE TITLE
initializeCommand can block VS Code if it prompts for user input

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode-remote-release#2434]
Testing #2407

- VSCode Version:
- Local OS Version:
- Remote OS Version:
- Remote Extension/Connection Type: SSH/Docker/WSL

Steps to Reproduce:

1. (mistakenly) Create a `devcontainer` that sets a `initializeCommand` command that prompts for user input: 

    ```
	"initializeCommand": "npm init"
    ```

2. Open the dev container

**Bug**

This blocks the container from ever starting properly.

**Comment by mjbvz** [comment]
Not sure what the expected behavior here is. Is there a way to run scripts in a no user input mode for example?

**Comment by chrmarti** [comment]
Not sure what we can do here in general. Maybe the answer is to run the command in a way that does not require input. E.g., `npm init -y`.

**Comment by mjbvz** [comment]
I don't think we can fully fix the problem. The main thing is that the user experience for a bad `initializeCommand` is quite poor today: vscode just seems to stall and it's difficult to exit out of this state

**Comment by chrmarti** [comment]
We can investigate connecting user input in the log terminal to the currently running command. Also with that the user might still want to change the command to not require any input.

I'm not sure we can detect when a command is waiting for user input. We could then automatically reveal the log terminal.
